coalals/utils/files.py

```diff
@@ -0,0 +1,110 @@
+from os.path import isabs, dirname
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+class UriUtils:
+
+    @staticmethod
+    def path_from_uri(uri):
+        if not uri.startswith('file://'):
+            return uri
+
+        _, path = uri.split('file://', 1)
+        return path
+
+    @classmethod
+    def dir_from_uri(cls, uri):
+        return dirname(cls.path_from_uri(uri))
+
+    @staticmethod
+    def file_to_uri(filename):
+        return 'file://{0}'.format(filename)
+
+
+class FileProxy:
```

Comment:
Maybe we could add doc for the class since it may conflicts ppl who are not familiar with the project.